benchmarks/benchmarks/ci/esmf_regridder.py

```diff
@@ -112,7 +112,7 @@ def setup_cache(self):
         n_lats_src = 200
         n_lons_tgt = 20
         n_lats_tgt = 40
-        h = 100
+        h = 2000
```

Comment:
Size of the source cube is increased since iris has a minimum size of chunk when loading a cube. If the source cube isn't large enough, there will only be one chunk and parallelisation cannot be demonstrated.